### PR TITLE
added translation for wallet and learning PSA

### PR DESCRIPTION
**Pull request by netrik182**

### CODE REVIEW

**Comment by lh1008** [comment]
Thank you @netrik182 :+1: 

Pinging @botoserelepe or @raiting89us for review. Please let me know who will do the review to add you to the taiga assigned tasks. :)

**Comment by botoserelepe** [comment]
@lh1008 I can also do this one :)

**Comment by lh1008** [comment]
Great @botoserelepe will add this review on the assign tasks. Thanks! :)

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Uma carteira segura é uma carteira feliz
+
+Sua carteira Monero (XMR) é como um cofre que guarda seus XMR e os mantém seguros. Se você quer ter certeza que isso se mantenha dessa forma, existem algumas boas práticas que você poderá seguir para manter sua carteria o mais segura possível.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]:
```suggestion
Sua carteira Monero (XMR) é como um cofre que guarda seus XMR e os mantém seguros. Se você quer ter certeza que isso se mantenha dessa forma, existem algumas boas práticas que você poderá seguir para manter sua carteira o mais segura possível.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Carteiras "quentes" são carteiras que estão conectadas na internet, enquanto carteiras "frias" não. Geralmente as carteiras frias são mais seguras, mas ela sacrificam a conveniência.
+
+Uma carteira "hard", como as populares Ledger e Trezor podem ser vistas como uma mistura entre carteiras quentes e frias pois você pode conecta-la em um dispositivo com internet para enviar ou receber XMR, mas uma vez desconectadas elas não serão acessíveis via internet.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]:
```suggestion
As carteiras "hard", como as populares Ledger e Trezor, podem ser vistas como uma mistura entre carteiras quentes e frias, pois você pode conectá-las a um dispositivo com internet para enviar ou receber XMR, mas uma vez desconectadas elas não serão mais acessíveis via internet.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+### Aqui vão algumas boas práticas gerais para se ter conhecimento quanto a segurança das carteiras:
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]:
```suggestion
### Aqui vão algumas boas práticas gerais para se ter conhecimento quanto à segurança das carteiras:
```

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+2. Se estiver configurando uma carteira fria (por exemplo uma carteira "em papel") para guardar a longo prazo, faça sua pesquisa e siga um dos tutoriais online que têm os comentários e as avaliações mais positivas. Como nestes links do Reddit:
+
+  a. Tutorial de como enviar para uma carteira em papel (Sending to Paper Wallet Tutorial)
+r/Monero/comments/7t5tyx/tutorial_sending_xmr_to_a_paper_wallet_generated
+
+  b. Guia para gerar uma carteira fria em papel em segurança (Guide for Securely Generating an Cold Paper Wallet)
+r/Monero/comments/48cgmd/an_extensive_guide_for_securely_generating_an
+
+3. Considere manter um carteira fria para os XMR que você pretende guardar por um tempo e uma carteira fria para os XMR que você pretende gastar ou negociar. Dica: Carteira fria=conta poupança, Carteira quente=conta corrente.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]:
```suggestion
3. Considere manter uma carteira fria para os XMR que você pretende guardar por um longo tempo e uma carteira quente para os XMR que você pretende gastar ou negociar com maior frequência. Dica: Carteira fria = conta poupança, Carteira quente = conta corrente.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+4. Visite fóruns online e pergunte para aprender as diferentes maneiras que as pessoas mantém suas chaves privadas ou sementes mnemônicas protegidas e seguras, depois escolha a opção que melhor funcione para você. Comece aqui: https://getmonero.org/community/hangouts/. Tem alguma estratégia é melhor do que não ter nenhuma. Já percebeu que pesquisar é um tema importante?
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]:
```suggestion
4. Visite fóruns online para se informar sobre as diferentes maneiras que as pessoas usam para manter suas chaves privadas ou sementes mnemônicas protegidas e seguras, e, a seguir, escolha a maneira que melhor funcione para você. Comece aqui: https://getmonero.org/community/hangouts/. Ter alguma estratégia é melhor do que não ter nenhuma. Já percebeu que pesquisar e se informar sobre o assunto é algo importante?
```

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+5. Siga outras boas práticas por aqui como nunca compartilhar suas chaves privadas, usar somente corretoras com boa reputação, manter suas carteiras atualizadas após cada atualização da rede, não falar sobre quanto XMR você tem e sempre busque por mais conhecimento!
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]:
```suggestion
5. Siga outras boas práticas como nunca compartilhar suas chaves privadas, usar somente corretoras com boa reputação, manter suas carteiras atualizadas após cada atualização da rede, não falar sobre quantos XMR você tem e manter-se sempre informado!
```

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md (hunk 5 above)
Comment (netrik182) [reply monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]: Added some of your suggestions and mixed others. Thank you.

**Comment by lh1008** [comment]
@rating89us the changes were added. Can you take a look and approve if it's ready? :)

File: monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+# Busque por mais conhecimento
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md]:
```suggestion
# Busque mais conhecimento
```

File: monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+
+No momento que escrevo esse texto, uma suposição que se faz é de que a maioria da comunidade Monero é composta por prioneiros da tecnologia, sendo assim mais experientes tecnicamente do que uma pessoa comum. O código do Monero vem sendo desenvolvido por algumas centenas de programadores, quantia que é muito maior do que a maioria dos projetos de cripto moedas. Especula-se que uma das razões pelas quais o preço da Monero não se aproxima ao do Bitcoin é porque a tecnologia por trás do Monero é difícil de entender e apreciar. Mas nós, os pioreiros, continuamos a avançar na esperança de que o Monero se tornará amplamente reconhecido e adotado. O Monero daqui a 100 anos ainda dependerá de pessoas, pessoas como você, trabalhando para proteger o ecossistema e avançar ainda mais o código. Esses são todos os motivos pelos quais recomendamos que as pessoas sempre busquem por conhecimento, ainda mais se elas não forem desenvolvedores. Você pode fortalecer sua "alfabetização sobre Monero" consumindo o maior número de informações que puder.
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md]:
```suggestion
No momento em que esse texto é escrito, uma suposição que se faz é de que a maioria da comunidade Monero é composta por prioneiros da tecnologia, sendo assim mais experientes tecnicamente do que uma pessoa comum. O código do Monero vem sendo desenvolvido por algumas centenas de programadores, quantia que é muito maior do que a maioria dos projetos de criptomoedas. Especula-se que uma das razões pelas quais o preço da Monero não se aproxima ao do Bitcoin é porque a tecnologia por trás do Monero é difícil de entender e apreciar. Mas nós, os pioneiros, continuamos a avançar na esperança de que o Monero se tornará amplamente reconhecido e adotado. O Monero daqui a 100 anos ainda dependerá de pessoas, pessoas como você, trabalhando para proteger o ecossistema e avançar ainda mais o código. Esses são todos os motivos pelos quais recomendamos que as pessoas sempre busquem conhecimento, ainda mais se elas não forem desenvolvedores. Você pode fortalecer sua "alfabetização sobre Monero" absorvendo o maior número de informações que puder.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+Ao aprender mais sobre os recursos do Monero, seu código, o espírito código aberto e cypherpunk que o originou e maneiras de melhorar sua segurança e sua privacidade ao utilizá-lo, juntos, poderemos fortalecer ainda mais o ecosistema do Monero. Ao ter um bom entendimento de todos os aspéctos do Monero, você também será capaz de falar sobre ele e ajudará a desenvolver uma maior adoção. Você se sentirá mais confortável e mais confiante em armazenar, usar e transacionar XMR. Conhecimento é poder e ele pode mudar o mundo. Sempre busque por mais conhecimento... e faça sua própria pesquisa!
```
Comment (botoserelepe) [inline monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md]:
```suggestion
Ao aprender mais sobre os recursos do Monero, seu código, o espírito do código aberto e do cypherpunk que o originou e maneiras de melhorar sua segurança e sua privacidade ao utilizá-lo, juntos, poderemos fortalecer ainda mais o ecossistema do Monero. Ao ter um bom entendimento de todos os aspectos do Monero, você também será capaz de falar sobre ele e ajudará a incentivar uma maior adoção da moeda. Você se sentirá mais confortável e mais confiante em armazenar, usar e transacionar XMR. Conhecimento é poder e ele pode mudar o mundo. Sempre busque mais conhecimento... e faça sua própria pesquisa!
```

File: monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md [hunk 10]
```diff
@@ -0,0 +1,18 @@
+# Busque mais conhecimento
+
+No momento em que esse texto é escrito, uma suposição que se faz é de que a maioria da comunidade Monero é composta por prioneiros da tecnologia, sendo assim mais experientes tecnicamente do que uma pessoa comum. O código do Monero vem sendo desenvolvido por algumas centenas de programadores, quantia que é muito maior do que a maioria dos projetos de criptomoedas. Especula-se que uma das razões pelas quais o preço da Monero não se aproxima ao do Bitcoin é porque a tecnologia por trás do Monero é difícil de entender e apreciar. Mas nós, os pioneiros, continuamos a avançar na esperança de que o Monero se tornará amplamente reconhecido e adotado. O Monero daqui a 100 anos ainda dependerá de pessoas, pessoas como você, trabalhando para proteger o ecossistema e avançar ainda mais o código. Esses são todos os motivos pelos quais recomendamos que as pessoas sempre busquem conhecimento, ainda mais se elas não forem desenvolvedores. Você pode fortalecer sua "alfabetização sobre Monero" absorvendo o maior número de informações que puder.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md]:
```suggestion
Atualmente assume-se que a maioria da comunidade Monero é composta por pessoas pioneiras em tecnologia, ou seja, usuários mais experientes tecnicamente do que uma pessoa leiga. O código do Monero vem sendo desenvolvido por centenas de programadores, quantia que é muito maior do que a maioria dos projetos de outras criptomoedas. Especula-se que uma das razões pelas quais o preço do Monero não se aproxima ao do Bitcoin é porque a sua tecnologia é difícil de se entender e apreciar. Mas nós, os pioneiros, continuamos em frente, na esperança de que um dia o nosso projeto se torne amplamente reconhecido e adotado. Daqui a 100 anos, o Monero ainda dependerá de pessoas, de pessoas como você, trabalhando para proteger o ecossistema e avançar ainda mais o código. Esses são os motivos pelos quais recomendamos que as pessoas sempre busquem conhecimento, especialmente se elas não forem desenvolvedores. Para fortalecer sua "alfabetização em Monero", absorva o maior número de informações que puder.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+
+Ao aprender mais sobre os recursos do Monero, seu código, o espírito do código aberto e do cypherpunk que o originou e maneiras de melhorar sua segurança e sua privacidade ao utilizá-lo, juntos, poderemos fortalecer ainda mais o ecossistema do Monero. Ao ter um bom entendimento de todos os aspectos do Monero, você também será capaz de falar sobre ele e ajudará a incentivar uma maior adoção da moeda. Você se sentirá mais confortável e mais confiante em armazenar, usar e transacionar XMR. Conhecimento é poder e ele pode mudar o mundo. Sempre busque mais conhecimento... e faça sua própria pesquisa!
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md]:
```suggestion
Ao aprendermos mais sobre os recursos do Monero, o seu código, o espírito do código aberto, a filosofia _cypherpunk_ que o originou e as maneiras de melhorar sua segurança e sua privacidade, fortalecemos juntos, ainda mais, todo o seu ecossistema. Se você tiver um bom entendimento de todos os aspectos do Monero, você também será capaz de participar de discussões com a comnuidade, assim como estará ajudando a incentivar uma adoção ainda maior da moeda. Você se sentirá mais confortável e mais confiante ao armazenar, receber e enviar XMR. Conhecimento é poder e ele pode mudar o mundo. Sempre busque mais conhecimento... e sempre faça sua própria pesquisa!
```

File: monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+
+### O futuro começa com você, aqui estão algumas leituras que poderão te ajudar a aprender mais sobre o Monero:
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md]:
```suggestion
### O futuro começa com você! Aqui estão algumas leituras que poderão lhe ajudar a aprender mais sobre o Monero:
```

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+
+Sua carteira Monero (XMR) é como um cofre que guarda seus XMR e os mantém seguros. Se você quer ter certeza que isso se mantenha dessa forma, existem algumas boas práticas que você poderá seguir para manter sua carteira o mais segura possível.
+
+Carteiras "quentes" são carteiras que estão conectadas na internet, enquanto carteiras "frias" não. Geralmente as carteiras frias são mais seguras, mas ela sacrificam a conveniência.
+
+As carteiras "hard", como as populares Ledger e Trezor podem ser vistas como uma mistura entre carteiras quentes e frias pois você pode conecta-la em um dispositivo com internet para enviar ou receber XMR, mas uma vez desconectadas elas não serão acessíveis via internet.
+
+### Aqui vão algumas boas práticas gerais para se ter conhecimento quanto à segurança das carteiras:
+
+1. Para evitar golpes que eventualmente roubem seus XMR, somente use carteiras aprovadas pela comunidade Monero. Um bom ponto de partida é a seção de informações do r/monero que pode ser acessada aqui: https://www.reddit.com/r/Monero/.
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]:
```suggestion
1. Para evitar alguns golpes que tentam roubar seus XMR, somente use carteiras aprovadas pela comunidade Monero. Um bom ponto de partida é a seção de informações do r/monero que pode ser acessada aqui: https://www.reddit.com/r/Monero/.
```

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md [hunk 14]
```diff
@@ -0,0 +1,25 @@
...
+r/Monero/comments/7t5tyx/tutorial_sending_xmr_to_a_paper_wallet_generated
+
+  b. Guia para gerar uma carteira fria em papel em segurança (Guide for Securely Generating an Cold Paper Wallet)
+r/Monero/comments/48cgmd/an_extensive_guide_for_securely_generating_an
+
+3. Considere manter uma carteira fria para os XMR que você pretende guardar por um longo tempo e uma carteira quente para os XMR que você pretende gastar ou negociar com maior frequência. Dica: Carteira fria = conta poupança, Carteira quente = conta corrente.
+
+4. Visite fóruns online para aprender as diferentes maneiras que as pessoas usam para manter suas chaves privadas ou sementes mnemônicas protegidas e seguras, depois escolha a opção que melhor funcione para você. Comece aqui: https://getmonero.org/community/hangouts/. Ter alguma estratégia é melhor do que não ter nenhuma. Já percebeu que pesquisar e se informar é algo importante?
+
+5. Siga outras boas práticas como a de nunca compartilhar suas chaves privadas, usar somente corretoras com boa reputação, manter suas carteiras atualizadas após cada atualização da rede, não falar sobre quantos XMR você tem e sempre buscar por mais conhecimento!
```
Comment (rating89us) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]:
```suggestion
5. Siga outras boas práticas como a de nunca compartilhar suas chaves privadas, usar somente corretoras com boa reputação, manter suas carteiras atualizadas após cada atualização da rede, não falar sobre quantos XMR você tem e sempre buscar mais conhecimento!
```

File: monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md [hunk 15]
```diff
@@ -0,0 +1,18 @@
...
+
+Ao aprender mais sobre os recursos do Monero, seu código, o espírito código aberto e cypherpunk que o originou e maneiras de melhorar sua segurança e sua privacidade ao utilizá-lo, juntos, poderemos fortalecer ainda mais o ecosistema do Monero. Ao ter um bom entendimento de todos os aspéctos do Monero, você também será capaz de falar sobre ele e ajudará a desenvolver uma maior adoção. Você se sentirá mais confortável e mais confiante em armazenar, usar e transacionar XMR. Conhecimento é poder e ele pode mudar o mundo. Sempre busque por mais conhecimento... e faça sua própria pesquisa!
+
+### O futuro começa com você, aqui estão algumas leituras que poderão te ajudar a aprender mais sobre o Monero:
+
+**_[getmonero.org (Site oficial)](https://getmonero.org/)_**
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt-br/PSA/files/learning/learning_pt-br.md]: Just noticed the getmonero.org site is now available in Brazilian Portuguese (pt-br). Now is a good chance to add the Portuguese links to the translations, so I will suggest changing starting with these ones. And this section needs to add a (+) sign at the beginning of every link to be a list.

Links to change:
- https://www.getmonero.org/pt-br/
- https://www.getmonero.org/pt-br/get-started/faq/

And by adding the (+) it should look like this:
+ **_[getmonero.org (Site oficial)](https://www.getmonero.org/pt-br/)_**
+ **_[monero.how](https://www.monero.how/)_**
+ **_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
+ **_[Guia do Monero (postagem)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
+ **_[Introdução técnica ao Monero](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
+ **_[Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf)_**
+ **_[Golpes a serem evitados](https://www.reddit.com/r/Monero/wiki/avoid)_**
+ **_[Monero Perguntas Frequêntes](https://www.getmonero.org/pt-br/get-started/faq/)_**
+ **_[Conecte-se com a Comunidade Monero](https://getmonero.org/community/hangouts/)_**
**_[Mastering Monero](https://masteringmonero.com/)_**

File: monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md (hunk 5 above)
Comment (lh1008) [inline monero-outreach-docs/translations/pt-br/PSA/files/wallet/wallet_pt-br.md]: This link can also be changed:

https://www.getmonero.org/pt-br/community/hangouts/

**Pull request merged**